src/fastertransformer/triton_backend/m2m/M2MTritonModel.cc

```diff
@@ -61,12 +61,8 @@ M2MTritonModel<T>::M2MTritonModel(INIReader reader, std::string model_dir): mode
     encoder_num_layer_     = reader.GetInteger("encoder", "num_layers");
     encoder_vocab_size_    = reader.GetInteger("encoder", "vocab_size");
     encoder_max_pos_seq_len_ = reader.GetInteger("encoder", "max_pos_seq_len");
-    mbart_para_              = reader.GetBoolean("encoder", "mbart", false);
-    if (mbart_para_) {
-        layernorm_type_ = ft::LayerNormType::pre_layernorm;
-    } else {
-        layernorm_type_ = ft::LayerNormType::post_layernorm;
-    }
+    layernorm_type_ = ft::LayerNormType::pre_layernorm;
```

Comment:
is this still needed since. you said no layer norm

Reply:
We still do layernorm, just not the one before the transformer block (on the embedding lookup). I just removed the layernorm_type though to simplify the codebase